src/app/components/PortraitVideoCarousel/PortraitCarouselNavigation/index.tsx

```diff
@@ -0,0 +1,81 @@
+/** @jsx jsx */
+import { useCallback, useContext, useEffect, useState } from 'react';
+import { jsx } from '@emotion/react';
+import { NavigationButtonsProp } from '#app/models/types/portraitVideo';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import styles, { PROMO_ITEM_WIDTH } from './index.styles';
+import { LeftChevron, RightChevron } from '../../icons';
+
+const defaultTranslations = {
+  scrollLeftAria: 'Scroll left',
+  scrollRightAria: 'Scroll right',
```

Comment:
Let's not include the word aria in the translation - it will be mainly editorial updating these values in the service file, so `scrollLeft` and `scrollRight` is better
```suggestion
  scrollLeft: 'Scroll left',
  scrollRight: 'Scroll right',
```